modules/API/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "description": "The EVerest API module, exposing some internal functionality on an external MQTT connection.",
+    "config": {
+    },
+    "provides": {
+        "main": {
+            "description": "EVerest API",
+            "interface": "empty"
```

Comment:
We could implement the complete logic in the module and don't provide anything

Reply:
That would be great, however our tooling doesn't really support modules that do not provide at least one empty interface at the moment

Reply:
Isn't that only the schema requirement?

Reply:
Yes, but isn't that just a technicality? I can't create a new module with

1. an empty provides object in the manifest
2. without a provides entry in the manifest

We could certainly remove this requirement from the schema, but we should check that it doesn't break any assumptions in everest-framework

Reply:
I see, i wouldn't impose this requirement - because we shouldn't force a module to provide something. On the other hand it should also be possible to only provide something and not to require something, which is already possible.
But if we don't require AND provide anything, then it might look a bit strange

Reply:
I agree, we should absolutely change this in everest-framework.
One could certainly imagine some module that doesn't interact with any other modules but still does some external mqtt communication. Such a module probably would not require AND provide anything. Might not be too useful, but maybe there are certain specialized use cases for that.